railties/lib/rails/generators/rails/app/templates/bin/yarn.tt

```diff
@@ -2,7 +2,7 @@ APP_ROOT = File.expand_path("..", __dir__)
 Dir.chdir(APP_ROOT) do
   yarn = ENV["PATH"].split(File::PATH_SEPARATOR).
     select { |dir| File.expand_path(dir) != __dir__ }.
-    product(["yarn", "yarn.cmd", "yarn.ps1"]).
+    product(["yarn", "yarnpkg", "yarn.cmd", "yarn.ps1"]).
```

Comment:
@BerkhanBerkdemir Can we put it at the end? 🤔

Reply:
My reasoning I put `yarnpkg` in the second place because it can be found in classic Yarn installation and the last two is related with Windows systems. Welcome to hear your thoughts, @zzak.

Reply:
I guess I could resolve this question by myself, but I need more thoughts from other project members.